### ISSUE TITLE
Descobrir e automatizar o processo de geração do JAR

### ISSUE THREAD

**Issue by gabriel-francischini** [watanabe9090/Controle_de_estoque#9]
Em teoria, seria só gerar o JAR e passar para o pessoal que funcionaria.
A gente precisa pegar essa teoria e colocar na prática:

Quais são os comandos/processos/botões para gerar o JAR?
Quais coisas o usuário precisa instalar na máquina dele pra rodar o nosso programa? O Java? O nosso zip? Um navegador web?

Seria uma boa criar uma VM leve, pode ser uma VM com o Windows 7 ou o Windows 10, e instalar o nosso software em uma máquina toalmente _clean_. É natural que o software vai rodar nos nossos computadores com IntelliJ, vários pacotes do Maven instaldo, múltiplas versões do java... mas será que roda na máquina de um cliente (aluno ou professora) que não tem nada disso instalado?

Talvez seria uma boa escrever os requisitos mínimos (de software e hardware) em algum papel, pois coisas que são óbvias e cotidianas para a gente como um navegador web, não funcionam no Windows XP 3 para trás (que ainda é usado em muita instituições por aí). Mas como estamos com pouco tempo, basta ter um processo de gerar o JAR que consiga rodar numa instalação mínima/limpa do Windows 7 ou Windows 10 que já vai garantir que o nosso software rode sem problemas no dia da apresentação

**Comment by watanabe9090** [comment]
Verdade, quanto a compatibilidade eu não sei se vai ser possível garantir sobre muitas versões, então a gente vai ter que "chutar alto", mas a respeito do JAR, a gente pode testar em algumas VMs.
Atualmente nós estamos desenvolvendo em cima da versão 11
